packages/at_onboarding_flutter/lib/screen/at_onboarding_reset_result.dart

```diff
@@ -0,0 +1,4 @@
+enum AtOnboardingResetResult {
+  cancelled,
+  success,
+}
```

Comment:
@HamdaanAliQuatil screen may not be the right folder to keep **AtOnboardingResetResult**.
Can we move `AtOnboardingResetResult ` in `at_onboarding_result.dart` file ?